theme.json

```diff
@@ -97,10 +97,10 @@
       "title": "Order Cancel"
     },
     {
-      "id": "item.cancelled",
+      "id": "shipment.itemscanceled",
       "properties": {},
       "template": "email/item-cancel",
-      "title": "Item Cancel"
+      "title": "Shipment Item Canceled"
```

Comment:
Just name it _Item Canceled_

Reply:
As suggested, I have updated the title to **Item Canceled**